protocols/dcutr/src/handler/relayed.rs

```diff
@@ -257,10 +257,6 @@ impl ConnectionHandler for Handler {
             return KeepAlive::Yes;
         }
```

Comment:
We shouldn't actually need to check for `queued_events` because we don't call `connection_keep_alive` whilst the handler is returned `Poll::Ready` so you can remove the above (either in this PR or a separate one, up to you!)

Reply:
removed 3e75728